src/test/java/teammates/test/cases/automated/SubmissionsAdjustmentTest.java

```diff
@@ -218,8 +218,11 @@ public void testEnrollStudentsWithScheduledSubmissionAdjustment() throws Excepti
         } catch (EnrollException e) {
             String actualErrorMessage = e.getLocalizedMessage();
 
-            String errorReason = String.format(FieldValidator.EMAIL_ERROR_MESSAGE, "e6@g@",
-                    FieldValidator.REASON_INCORRECT_FORMAT);
+            String errorReason = FieldValidator.EMAIL_ERROR_MESSAGE
+                                     .replace("${userInput}", "e6@g@")
+                                     .replace("${fieldName}", FieldValidator.EMAIL_FIELD_NAME)
+                                     .replace("${reason}", FieldValidator.REASON_INCORRECT_FORMAT)
+                                     .replace("${maxLength}", String.valueOf(FieldValidator.EMAIL_MAX_LENGTH));
```

Comment:
Granted that the reason for keeping the `getPopulatedErrorMessage` is valid, I don't really like repetitions of this procedure. Let's use reflection here instead. Add this method to `BaseTestCase.java`:
```java
protected static String getPopulatedErrorMessage(String messageTemplate, String userInput, String fieldName,
                                                 String errorReason, int maxLength) {
    try {
        Method getPopulatedErrorMessage = FieldValidator.class.getDeclaredMethod("getPopulatedErrorMessage",
                String.class, String.class, String.class, String.class, int.class);
        getPopulatedErrorMessage.setAccessible(true);
        return (String) getPopulatedErrorMessage.invoke(null, messageTemplate, userInput, fieldName,
                                                        errorReason, maxLength);
    } catch (Exception e) {
        throw new RuntimeException(e);
    }
}
```
And you can call `getPopulatedErrorMessage` from any test case without breaking the abstraction. Do the same with the 4-params version.